### PR TITLE
Add CODEOWNERS file

### PR DESCRIPTION
**Pull request by DarkEld3r**
Open questions:
- Should all related people be involved into a review or only the minimal amount?
- Currently some modules are left without dedicated reviewers.
- Volunteers are welcome. :upside_down_face:

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+examples/cryptocurrency-advanced/* @darkeld3r @aleksuss @vitvakatu @ilammy
+
+# Timestemping:
+examples/timestemping/* @darkeld3r @aleksuss @ozkriff @mvlabat @vitvakatu
+
+# Configuration service:
+services/* @darkeld3r @aleksuss @alekseysidorov @vldm @slowli
+
+# Time-oracle service:
+testkit/* @darkeld3r @aleksuss @alekseysidorov @slowli
```
Comment (vitvakatu) [inline .github/CODEOWNERS]: I think there is a mistake here (duplicated line)

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+# Cryptocurrency examples:
+examples/cryptocurrency/* @darkeld3r @aleksuss @vitvakatu @ilammy
+examples/cryptocurrency-advanced/* @darkeld3r @aleksuss @vitvakatu @ilammy
+
+# Timestemping:
+examples/timestemping/* @darkeld3r @aleksuss @ozkriff @mvlabat @vitvakatu
+
+# Configuration service:
+services/* @darkeld3r @aleksuss @alekseysidorov @vldm @slowli
```
Comment (vitvakatu) [inline .github/CODEOWNERS]: This is also wrong

File: .github/CODEOWNERS [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+# Cryptography:
+crypto/* @darkeld3r @aleksuss @pavel-mukhanov @vldm @slowli
+
+# Cryptocurrency examples:
+examples/cryptocurrency/* @darkeld3r @aleksuss @vitvakatu @ilammy
+examples/cryptocurrency-advanced/* @darkeld3r @aleksuss @vitvakatu @ilammy
+
+# Timestemping:
+examples/timestemping/* @darkeld3r @aleksuss @ozkriff @mvlabat @vitvakatu
```
Comment (vitvakatu) [inline .github/CODEOWNERS]: timestAmping?

File: .github/CODEOWNERS [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+exonum/src/api/* @darkeld3r @aleksuss @alekseysidorov @andreyluchin
+
+# Encoding:
+exonum/src/encoding/* @darkeld3r @aleksuss @vldm
+
+# Events:
+exonum/src/events/* @darkeld3r @aleksuss @alekseysidorov @andreyluchin @ilammy
+
+# Node (consensus):
+exonum/src/events/* @darkeld3r @aleksuss @slowli
```
Comment (vldm) [inline .github/CODEOWNERS]: node behaviour and consensus algorithm is located in `exonum/src/node`

File: .github/CODEOWNERS [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+exonum/benches/* @darkeld3r @aleksuss @vldm @alekseysidorov @andreyluchin
+
+# Web api:
+exonum/src/api/* @darkeld3r @aleksuss @alekseysidorov @andreyluchin
+
+# Encoding:
+exonum/src/encoding/* @darkeld3r @aleksuss @vldm
+
+# Events:
+exonum/src/events/* @darkeld3r @aleksuss @alekseysidorov @andreyluchin @ilammy
```
Comment (pavel-mukhanov) [inline .github/CODEOWNERS]: Pls add me to events, because I working a lot with network :)

Comment (DarkEld3r) [reply .github/CODEOWNERS]: Done: https://github.com/exonum/exonum/pull/938

**Pull request merged**